book/website/reproducible-research/rdm/rdm-data-curation.md

```diff
@@ -0,0 +1,26 @@
+(rr-rdm-data-curation)=
+# Data Curation
+Data curation is one of the key aspects of RDM. It  involves capturing of research data, appraisal, disposal, description, preservation, access, reuse and transformation. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for.  Data curation  enables you to easily  access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine if you don't follow the data curation process, it becomes difficult to access that data.
```

Comment:
Please spell out RDM on the first line, then RDM in brackets.

Reply:
```suggestion
Data curation is one of the key aspects of Research Data Management (RDM). It  involves capturing of research data, appraisal, disposal, description, preservation, access, reuse and transformation. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for.  Data curation  enables you to easily  access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine if you don't follow the data curation process, it becomes difficult to access that data.
```

Suggesting the change mentioned by Emma. You can commit changes if this is right.

Reply:
Resolved by Jen's edits.